examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>May you run a backtest after each method separately, the logic this suffixes might be added while merging the dataframes with method <code>merge</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>This method works only for two dataframes. I think concatenating all at once and adding suffixes on the fly will be clearer.</p><p>&nbsp;</p>